### PR TITLE
Update Katib images on 0.3 branch

### LINKED ISSUE #1604: Update Katib image in 0.3 branch
Tracking bug for updating the Katib images on the 0.3 branch

#1597 updates it on master. Will need to cherry pick it onto the release branch.

### PR DESCRIPTION
**Pull request by jlewi**
Fix #1604  update Katib image on 0.3. branch

* Remove katib releaser its not used any more; we push on postsubmit.

* Add a script to update Docker images in Katib ksonnet component
  and use it to upgrade Katib ksonnet components.

* Update the release instructions

  * Add instructions for Katib
  * Remove bootstrapper instructions; its no longer part of releases.

Related to kubeflow/katib#79

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1605)

### CODE REVIEW

File: docs_dev/releasing.md [hunk 1]
```diff
@@ -262,6 +251,20 @@ the correct Docker image.
...
 
+### Katib
+
+1. Identify the tag of the Katib images to use
+
+   * Katib images should be pushed for each postsubmit
+
+1. Modify the script update_katib_ksonnet.sh 
+
+    * set REALEASE to the tag you want to use
```
Comment (richardsliu) [inline docs_dev/releasing.md]: Typo

**Comment by jlewi** [comment]
/assign @ankushagarwal 

@ankushagarwal Can you take a look please?

**Pull request merged**